### ISSUE TITLE
Copy to clipboard button after crash

### ISSUE THREAD

**Issue by koxx12-dev** [isXander/CrashHelper#2]
Button in that message popup that Copies hastebin url to clipboard
tray notifinaction saying `Copied crash log to clipboard` would be also cool

by tray i mean this _(random img from google)_
![image](https://user-images.githubusercontent.com/80456351/127871342-fa3f7c72-e757-464c-8257-4131ef7ae66d.png)

**Comment by isXander** [comment]
I don't see desktop toast notifications ever being a thing.

1. it's annoying
2. there is no good way to do it universally

however, copying to clipboard is trivial

**Comment by koxx12-dev** [comment]
> 2. there is no good way to do it universally
```java
public class TrayIconDemo { public static void main(String[] args) throws AWTException { if (SystemTray.isSupported()) { TrayIconDemo td = new TrayIconDemo(); td.displayTray(); } else { System.err.println("System tray not supported!"); } } public void displayTray() throws AWTException { //Obtain only one instance of the SystemTray object SystemTray tray = SystemTray.getSystemTray(); //If the icon is a file Image image = Toolkit.getDefaultToolkit().createImage("icon.png"); //Alternative (if the icon is on the classpath): //Image image = Toolkit.getDefaultToolkit().createImage(getClass().getResource("icon.png")); TrayIcon trayIcon = new TrayIcon(image, "Tray Demo"); //Let the system resize the image if needed trayIcon.setImageAutoSize(true); //Set tooltip text for the tray icon trayIcon.setToolTip("System tray icon demo"); tray.add(trayIcon); trayIcon.displayMessage("Hello, World", "notification demo", MessageType.INFO); } }
```
it should work on every os that supports it `idk if its formated i sent it from mobile`

**Comment by koxx12-dev** [comment]
and wdym by annoying

**Comment by isXander** [comment]
It's all one line

**Comment by koxx12-dev** [comment]
nice

**Comment by isXander** [comment]
Closed with a725163e4edeb2eae32dc4f0dfd4fb23a3bf774d
